raw/mock_exam/docs/questions/question-39.adoc

```diff
@@ -0,0 +1,90 @@
+// tag::DE[]
+=== Frage 39
+**ID: Q-20-04-36**
+
+[cols="2,8,2", frame=ends, grid=rows]
+|===
+| P-Frage: 
+| Wählen Sie die **drei** am besten passenden Antworten aus
+| 1 Punkt
+|===
+
+Sie versuchen, ihre Architektur quantitativ zu untersuchen.
+Welche der folgenden Größen können Sie für Ihre Softwarearchitektur zuverlässig messen?
+Wählen Sie die drei am besten passenden Antworten aus.
+
+[cols="1a,1,10", frame=none, grid=none]
+|===
+
+| {y}
+| (a)
+| Größe der Bausteine (z.{nbsp}B. Lines-of-Code).
+
+| {y}
+| (b)
+| Änderungsrate des Quellcodes der Komponenten.
+
+| {n}
+| (c)
+| Kohäsion der Architekturkomponenten.
+
+| {n}
+| (d)
+| Sicherheitsstufe einer Komponente.
+
+| {y}
+| (e)
+| Anzahl der Entwickler, die zu einer bestimmten Komponente beigetragen haben.
+
+|===
+
+// end::DE[]
+
+// tag::EN[]
+=== Question 39
+**ID: Q-20-04-36**
+
+[cols="2,8,2", frame=ends, grid=rows]
+|===
+| P-Question: 
+| Select the **three** best fitting answers
+| 1 point
+|===
+
+You try to quantitatively analyze your architecture.
+Which three of the following properties can you measure reliably in your software architecture?
+Pick the three best fitting answers.
+
+[cols="1a,1,10", frame=none, grid=none]
+|===
+
+| {y}
+| (a)
+| Size of building blocks (e.g. LOC).
+
+| {y}
+| (b)
+| Change rate of the source code of components.
+
+| {n}
+| (c)
+| Cohesion of the architectural components.
+
+| {n}
+| (d)
+| Security level of a component.
+
+| {y}
+| (e)
+| Number of the developers that contributed to a specific component.
+|===
+
+// end::EN[]
+
+// tag::EXPLANATION[]
+#### Explanation
```

Comment:
Erscheint auch in der deutschen Version

Reply:
das ist aktuell so - explanations sind nur einsprachig möglich.